packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -182,10 +192,32 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           ariaSelected: selectedIndices[0] === undefined ? undefined : true
         };
 
+    this._classNames = getClassNames(propStyles, {
+      theme,
+      className,
+      hasError: Boolean(errorMessage && errorMessage.length > 0),
+      isOpen,
+      required,
+      disabled,
+      isRenderingPlaceholder: !selectedOptions.length,
+      panelClassName: !!panelProps ? panelProps.className : undefined,
+      calloutClassName: !!calloutProps ? calloutProps.className : undefined
+    });
+
+    const labelStyles = this._classNames.subComponentStyles
+      ? (this._classNames.subComponentStyles.label as IStyleFunctionOrObject<ILabelStyleProps, ILabelStyles>)
```

Comment:
The `as` cast here is a Typescript 2.8 friction

Reply:
I have an idea how we can potentially avoid the null/undefined check. It would make it easier to consume subComponentStyles. WIll be addressed in a separate PR.